tsconfig.json

```diff
@@ -1,30 +1,48 @@
 {
+  "buildOnSave": false,
+  "compileOnSave": true,
   "compilerOptions": {
     "allowJs": false,
+    "baseUrl": "app/scripts/modules",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
+    "lib": ["es2016", "dom"],
     "moduleResolution": "node",
     "module": "commonjs",
-    "target": "es5",
     "noImplicitAny": true,
+    "outDir": "transpiled",
     "pretty": true,
     "removeComments": true,
-    "rootDirs": [
-      "app/scripts/modules"
-    ],
+    "rootDir": ".",
+    "skipLibCheck": true,
     "sourceMap": true,
-    "outDir": "transpiled"
+    "target": "es5",
+    "typeRoots": [
+      "node_modules/@types/"
+    ],
+    "types": [
+      "angular",
+      "angular-mocks",
+      "angular-ui-router",
+      "jasmine",
+      "jquery",
+      "lodash",
+      "moment-timezone",
+      "node"
+    ]
   },
-  "files": [
-    "typings/index.d.ts",
-    "test/helpers/customMatchers.d.ts"
-  ],
   "exclude": [
+    "**/*.spec.ts",
     "build",
     "node",
     "node_modules",
     "transpiled"
   ],
-  "compileOnSave": true,
-  "buildOnSave": false
+  "include": [
+    "app/scripts/modules/**/*.ts"
+  ],
+  "awesomeTypescriptLoaderOptions": {
+    "useBabel": true,
+    "useCache": true
+  }
```

Comment:
Nope, it was an artifact of some playing around w/ caching compiled TS files in ATL, but it will require moving to Webpack 2.0.  This was more than I wanted to bite off with this PR so I bailed on that for now.  I will remove this.